src/components/AdminPanelLayout/UserListItem.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {FormEvent, useState} from 'react';
+import {Role, User} from '../../types/User';
+
+export interface ChangeUserRoleRequestSchema {
```

Comment:
Move this to right before changeUserRole function in the api. Since it is the request interface, it should belong with the function that is making the request.